include/mgba/internal/gba/extra/extensions-ids.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2013-2021 Jeffrey Pfau
```

Comment:
This file should still be merged into `extensions.h`--there's no strong reason to keep it separate.

Reply:
Fixed.